.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Wed Aug 19 09:04:08 COT 2020
+gradle.version=6.4.1
```

Comment:
All .gradle should not be in the remote repository.

Reply:
add this to .gitignore

Reply:
.gitignore has been added and it is now working